app/lib/shared/exceptions.dart

```diff
@@ -78,11 +78,10 @@ class InvalidInputException extends ResponseException {
   }
 
   /// Throw [InvalidInputException] if [value] doesn't match [pattern].
-  static void checkMatchPattern(String value, String name, Pattern pattern) {
```

Comment:
Please match the method name and param (e.g. `checkMatchRegExp` or use `Pattern` instead of `RegExp`).

Reply:
Done